.github/CHANGELOG.md

```diff
@@ -39,6 +39,10 @@
 
 ### Documentation
 
+* The centralized [Xanadu Sphinx Theme](https://github.com/XanaduAI/xanadu-sphinx-theme)
+  is now used to style the Sphinx documentation.
+  [(#23)](https://github.com/XanaduAI/xanadu-cloud-client/pull/23)
+
```

Comment:
Minor point - @Mandrenkov are you listed as a contributor for this release?

Reply:
Good catch! 🤦‍♂️